habitat-lab/habitat/utils/common.py

```diff
@@ -30,6 +31,13 @@ def check_make_dir(directory_path: str) -> bool:
     return True
 
 
+def get_project_root() -> str:
```

Comment:
This will break if the location of this method changes or if the relative location of the data path changes. We need to fix this because I think it is very likely users will have a modified data path. Can you instead get the path from the object manager?

Reply:
Fixed this to get the path from config.